components/brave_wallet_ui/components/desktop/views/portfolio/index.tsx

```diff
@@ -224,7 +215,13 @@ const Portfolio = (props: Props) => {
   }, [selectedAsset, transactions])
 
   const fullAssetBalances = React.useMemo(() => {
-    return filteredAssetList.find((asset) => asset.asset.contractAddress.toLowerCase() === selectedAsset?.contractAddress.toLowerCase())
+    if (selectedAsset?.contractAddress === '') {
+      return filteredAssetList.find((asset) => asset.asset.symbol.toLowerCase() === selectedAsset?.symbol.toLowerCase())
+    }
+    return filteredAssetList.find(
+      (asset) =>
+        asset.asset.contractAddress.toLowerCase() === selectedAsset?.contractAddress.toLowerCase()
+    )
```

Comment:
What do you think of also checking for equality of chainId?

Reply:
Good call, added!